blog/2021-11-09-php-dead-code-detection.md

```diff
@@ -0,0 +1,24 @@
+# Dead Code Detection in PHP

```

Comment:
we will need a header for this along the lines of the other blogs :) you can copy & paste and just adjust the name / tags / time. I think `casestudy` is a good tag to add for this

Reply:
Yep. I'll add a header on the case study branch

Reply:
We should add this to the top level case study too - so that needs an extra paragraph